content/docs/connectors/oidc.md

```diff
@@ -56,7 +56,7 @@ connectors:
 
     # List of additional scopes to request in token response
     # Default is profile and email
-    # Full list at https://github.com/dexidp/dex/blob/master/Documentation/custom-scopes-claims-clients.md
+    # Full list at https://github.com/dexidp/website/blob/main/content/docs/custom-scopes-claims-clients.md
```

Comment:
Should we link to the site, https://dexidp.io/docs/custom-scopes-claims-clients/, rather than the source?

Reply:
Sure -
```suggestion
    # Full list at https://dexidp.io/docs/custom-scopes-claims-clients/
```
But I guess it won't appear as a link on the website though

Reply:
No, because it's in the code formatted section it won't appear as a link. 

Is there a place in [dexidp/dex repo](https://github.com/dexidp/dex) where this configuration sample comes from? I took a quick look, and didn't see it there, but if there is, we should probably change the link there too.